src/test.jl

```diff
@@ -0,0 +1,6 @@
+include("OHDSIAPI.jl")
+using .OHDSIAPI
+
+# OHDSIAPI.get_cohort_definition(1792865)
+
+OHDSIAPI.get_cohort_definition([1792865, 1792956])
```

Comment:
Here for testing, you should be able to use BrokenRecord.jl with the playback mechanism. You can take a look here: https://github.com/JuliaHealth/PubChemCrawler.jl/blob/e0a2100704f3a9beb189fb896fefc798e60a94d7/test/runtests.jl#L41

Reply:
I also think having a test for one and multiple is great.